### ISSUE TITLE
main.py第250行loss = criterion(outputs, y)-->loss = criterion(logits, y)

### ISSUE THREAD

**Issue by qianjinfighter** [MontaEllis/Pytorch-Medical-Segmentation#33]
你好，很感谢您的代码，我预测效果很差，多次修改损失函数都没有效果，看代码发现这里有点疑问，请问这里是不是错了？
man.py代码:
            logits = torch.sigmoid(outputs)
            labels = logits.clone()
            labels[labels>0.5] = 1
            labels[labels<=0.5] = 0
            loss = criterion(outputs, y)#有问题？应该为loss = criterion(logits, y)

**Comment by MontaEllis** [comment]
没有问题

**Comment by qianjinfighter** [comment]
啊 好的 谢谢
